appserver/admingui/plugin-service/src/main/java/org/glassfish/admingui/plugin/ConsolePluginService.java

```diff
@@ -48,35 +48,39 @@
 
 import jakarta.inject.Inject;
 
-
 /**
- *  <p>        This class provides access to {@link IntegrationPoint}s.</p>
+ * <p>
+ * This class provides access to {@link IntegrationPoint}s.
+ * </p>
  *
- *  @author Ken Paulsen        (ken.paulsen@sun.com)
+ * @author Ken Paulsen (ken.paulsen@sun.com)
  */
 @Service
 public class ConsolePluginService {
-    @Inject Logger logger;
-    @Inject ServiceLocator habitat;
-    @Inject IterableProvider<ConsoleProvider> providers;
-
-/*
-    @Inject ModulesRegistry modulesRegistry;
```

Comment:
I don't think the automatic formatting of all these classes helped. Maybe this commented out code could be even deleted.